content/en/blog/_posts/2021-0420-cncf-dan.md

```diff
@@ -0,0 +1,196 @@
++++ 
+title = "Thank You Dan Kohn" 
+date = 2021-03-19 
+author = ["Riaan Kleinhans"]
+lastmod = "Fri Mar 06 17:06:13 NZDT 2021" 
+summary = "He believed in ii, and in all of us." 
++++
+
+This is a story about relationships, and how they are the currency that moves
+this world forward. It is a story about ii's relationship with Dan Kohn, and how
```

Comment:
Perhaps, s/ii's/the ii team's/. Or, if "ii" is an initialism for <something> perhaps s/ii's/the <something> (ii) team's/

Perhaps also provide a little more background about what this team is or does a bit further on.








Maybe provide more background on what the ii team is or does a bit later on.